situp-core/src/integrationTest/java/com/amazon/situp/integration/EndToEndIntegrationTest.java

```diff
@@ -96,15 +98,10 @@ private void refreshIndices(final RestHighLevelClient restHighLevelClient) throw
         restHighLevelClient.indices().refresh(requestAll, RequestOptions.DEFAULT);
     }
 
-    private void sendExportTraceServiceRequestToSource(ExportTraceServiceRequest request) throws InvalidProtocolBufferException {
-        WebClient.of().execute(RequestHeaders.builder()
-                        .scheme(SessionProtocol.HTTP)
-                        .authority("127.0.0.1:21890")
-                        .method(HttpMethod.POST)
-                        .path("/opentelemetry.proto.collector.trace.v1.TraceService/Export")
-                        .contentType(MediaType.JSON_UTF_8)
-                        .build(),
-                HttpData.copyOf(JsonFormat.printer().print(request).getBytes())).aggregate().join();
+    private void sendExportTraceServiceRequestToSource(ExportTraceServiceRequest request) {
+        final TraceServiceGrpc.TraceServiceBlockingStub client = Clients.newClient(
```

Comment:
Looks like e2e failed here. Need to add `com.linecorp.armeria:armeria-grpc:1.0.0` into the dependencies of `integrationTest.gradle`

Reply:
already fixed.